timestamp-impl/src/test/java/com/palantir/timestamp/PersistentTimestampServiceTest.java

```diff
@@ -90,7 +91,7 @@ public void incrementUpperLimitIfOneMinuteElapsedSinceLastUpdate() throws Interr
         Thread.sleep(10);
         persistentTimestampService.getFreshTimestamp();
         Thread.sleep(10);
-        verify(timestampBoundStore, times(2)).storeUpperLimit(anyLong());
+        verify(timestampBoundStore, atLeast(2)).storeUpperLimit(anyLong());
```

Comment:
#466 should make this sadness go away (and it's unblocked now that DBKVS is in). 

The (soon-to-be-historical) reason this was set multiple times was that setting it happened in a busy-loop in a separate thread, so sometimes an extra call would sneak in. (That has no correctness implications because there is no harm in allocating more timestamps than necessary)